### PR TITLE
Refactored helm-chart-annotations.md

### PR DESCRIPTION
**Pull request by iamrajiv**
Formatted the file using Prettier and fixed grammatical mistakes.

### CODE REVIEW

File: docs/helm-chart-annotations.md [hunk 1]
```diff
@@ -3,6 +3,21 @@
...
+    lastCertifiedTime: 2021-04-22 10:49:29.714918174 +0000 UTC m=+1.932279870
+    certifiedOpenShiftVersions: "3.7.5"
+```
+
+# Certification Annotations
+
+## Verifier added annotations
+
+The chart-verifier tool adds annotations to a generated report, for example:
+
```
Comment (baijum) [inline docs/helm-chart-annotations.md:20]: This is a repetition of the below text

File: docs/helm-chart-annotations.md [hunk 2]
```diff
@@ -34,10 +50,33 @@ annotations:
...
+  charts.openshift.io/supportURL: http://acme-help-is-here.com/
+  charts.openshift.io/archs: x86_64
+```
+
+| Annotation                     | Description                                                              |
+| ------------------------------ | ------------------------------------------------------------------------ |
+| charts.openshift.io/provider   | Name of chart provider (e.g., Red Hat), ready to be displayed in UI.      |
+| charts.openshift.io/name       | Human readable chart name, ready to be displayed in UI.                  |
+| charts.openshift.io/supportURL | Where users can find information about the chart provider's support.     |
+| charts.openshift.io/archs      | Comma separated list of supported architectures (e.g., x86_64, s390x,...) |
```
Comment (baijum) [inline docs/helm-chart-annotations.md:82]: This is a repetition of the above text

**Review by baijum — CHANGES_REQUESTED** [review]
Please remove duplicate text and only keep relevant modifications.

**Comment by baijum** [comment]
@iamrajiv Sorry, I am a bit lost here. Can you please update the title and description with what is the intent of this pull request?

As of now, looks like some relevant content has been removed.

**Review by Srivaralakshmi — COMMENTED** [review]
Looks good, from my side.

**Review by baijum — APPROVED** [review]
LGTM

**Pull request merged**